redesign for lessons page

ifiguer opened the pull request:
Almost, forgot... This pull request is for issue 509: https://github.com/TheOdinProject/theodinproject/issues/509

KevinMulhern commented:
Hey @ifiguer, I checked this out locally and it looks nothing like the mock up? 😕 

There seems to be some parts missing. 

* The header at the top which has the course logo and course name
*  and the column at the left that contains the navigation for the lesson

Theres other things that should have been removed that are still there also.
* the breadcrumbs bar
* the adverts on the page

Whats going on?

ifiguer commented:
:( It is very likely that I did something wrong when sending the pull request, because it definitely works on my machine. I'll look into it when I get out of work. Thanks for being patient with me. I'm new to all this.

KevinMulhern commented:
Not problem, don't worry about it. You probably just forgot to commit your latest changes. It happens 😄

arun1595 commented on config/database.yml:
```diff
@@ -1,29 +0,0 @@
-# You may need to provide a username, in which case you'd use
```
Why is this file removed?

ifiguer replied:
My bad, I added it back

arun1595 commented on app/views/lessons/show.html.erb:
```diff
@@ -31,71 +12,45 @@
+        <h3>Solutions</h3>
       </div>
+
+      <h1 class="text-center">Got Questions?</h1>
```
Please use the `bottom_cta` partial. Have a look at [this](https://github.com/TheOdinProject/theodinproject/blob/redesign/app/views/static_pages/faq.html.erb#L10) for an example

ifiguer commented:
BTW do I need to make another pull request or are you able to see here the latest commits? I updated it to use the bottom_cta partial and added the correct classes to the buttons (button--primary).

KevinMulhern commented:
Hi @ifiguer I checked this out locally and its looks really good 😄 

The `database.yml` is still showing up in this PR. It shouldn't have to be changed for this. You can get it back to the way it was before and include it in this PR by doing this through git:
```
git checkout master config/database.yml
git commit --amend --no-edit
git push origin fix/redesign-lessons
```

Once thats done, this will be ready to merge.

Thank you for all your work on this one 😄

KevinMulhern commented:
Thanks @ifiguer 😄 
LGTM 👍 👍 👍

ifiguer commented:
Hi @KevinMulhern thanks for the very helpful instructions. I think we should be good now. One thing I should mention is that the buttons at the bottom (View Course, Next Lesson, etc.) aren't necessarily the best on smaller screens. I wasn't sure on how to best style them.

Also, the Mark Lesson Completed button works differently depending on whether the user is signed in (if they are not it redirects to the login page, and marks the lesson completed when the user is signed in) but I wasn't able to test if this worked properly. I'm hoping this was the intended behavior.

KevinMulhern commented:
No worries about those things @ifiguer 😄 

The mobile design for the lesson page hasn't been looked into yet, we will cover that in the next redesign review though.

Yeah thats the intended behaviour for those buttons 😄 

We have to do some follow up work to standardise the lesson content for all the lessons, the work you've done here has put us in a great position to do that. Thanks again for all your help we really do appreciate it ❤️ 

I'm about to go on a merging spree with these open PR's and deploy them to staging :shipit: 
keep an eye on https://gitter.im/TheOdinProject/Contributing to see when thats done so you can see your work on our staging server 😄
